UnitTests/PayPalTests/MockCheckout.swift

```diff
@@ -48,4 +49,4 @@ struct MockPayPalError: PayPalCheckoutErrorInfo {
     var reason: String
     var error: Error
 }
-// swiftlint:enable space_after_main_type
+// swiftlint:disable space_after_main_type
```

Comment:
Do we need this change?

Reply:
yeah both lines are disable, enabling at end

Reply:
Now we are disabling for any future structs/classes in this file instead of re-enabling the rule after these 2 struct's. I think we want to revert this to re-enable the rule again after these 2 struct's as expected.